package.json

```diff
@@ -163,27 +163,27 @@
                 {
                     "command": "extension.ccUndoCheckout",
                     "group": "cc@3",
-                    "when": "vscode-clearcase:enabled"
+                    "when": "editorTextFocus && !inOutput && vscode-clearcase:enabled"
```

Comment:
This is something I found somewhere in a vscode issue discussion talking about exact the same problem they had. So someone added this variable.